### PR TITLE
Fix compilcation errors : issue #1085

### PR DESCRIPTION
**Pull request by tosik**
This fixes the issue #1085

### CODE REVIEW

File: src/platform/qt/LoadSaveState.cpp [hunk 1]
```diff
@@ -13,6 +13,7 @@
 #include <QDateTime>
 #include <QKeyEvent>
 #include <QPainter>
+#include <QAction>
```
Comment (endrift) [inline src/platform/qt/LoadSaveState.cpp]: Please keep these in alphabetical order.

File: src/platform/qt/Window.h [hunk 2]
```diff
@@ -9,6 +9,7 @@
 #include <QList>
 #include <QMainWindow>
 #include <QTimer>
+#include <QAction>
```
Comment (endrift) [inline src/platform/qt/Window.h]: Ditto.

**Comment by endrift** [comment]
Closed due to fixing it myself after not getting a response

**Comment by tosik** [comment]
Sorry ...  I was sick.  Thank you.

**Comment by endrift** [comment]
Sorry, I got a bit impatient because I upgraded Qt and started hitting the build failure myself.

**Pull request closed without merging**